launcher/Flatpak.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-3.0-only
+/*
+ *  PolyMC - Minecraft Launcher
+ *  Copyright (C) 2022 dada513 <dada513@protonmail.com>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, version 3.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+#include <QFileInfo>
+
+namespace Flatpak
```

Comment:
Maybe you could just put this into Application.cpp, don't see a reason to have this in it's own namespace

Reply:
Initially I wanted this to be a separate namespace because I wanted to implement Portal dbus apis for drag and drop. I think they are too limited though so probably won’t do that and will move it to application